pkg/networkserver/redis/registry.go

```diff
@@ -152,26 +115,153 @@ func (r *DeviceRegistry) GetByEUI(ctx context.Context, joinEUI, devEUI types.EUI
 }
 
 type uplinkMatchSession struct {
-	LoRaWANVersion    ttnpb.MACVersion
```

Comment:
Any specific reason for re-ordering the fields here (and below)?

Reply:
Yes, this takes less space